src/library/playlisttablemodel.cpp

```diff
@@ -8,13 +8,14 @@
 
 #include "mixer/playermanager.h"
 
+/// \param preserveAll Whether to keep deleted tracks, default false
```

Comment:
I think this should go in the header file. And if you use param, you should use `@` since that's already used in our code base.

Reply:
you're totally right - about @ vs / I simply wasn't sure, will adjust